samples/DotNet/Microsoft.Azure.EventHubs/Rbac/CustomRole/readme.md

```diff
@@ -1,25 +1,25 @@
 # Receive events with the Event Processor Host in .NET Standard with a custom role which grants Listen claim for Event Hubs and blob claims for Storage accounts.
 
+**Note:** The sample in this folder uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the current library `Azure.Messaging.EventHubs`. This file has instructions for both.
 
-## Run the sample
-
-**Note:** This sample uses the legacy Event Hubs library `Microsoft.Azure.EventHubs`. We strongly recommend you to use the current library `Azure.Messaging.EventHubs`. 
+## Prerequisites
```

Comment:
I moved around the code here because the custom role set up instructions applies to both T1 and T2.